docs/topics/GATEWAY.md

```diff
@@ -247,7 +247,7 @@ Receiving payloads with the Gateway API is slightly more complex than sending. W
 
 ## Events
 
-### Ready
+### Ready - `READY`
```

Comment:
This breaks channel linking, I'd rather you note at the top how we generate the name then add it to all of these